includes/admin/class-amp-template-customizer.php

```diff
@@ -134,11 +134,16 @@ public function register_settings() {
 	 */
 	public function add_customizer_scripts() {
 		if ( ! amp_is_canonical() ) {
+			$asset_file   = AMP__DIR__ . '/assets/js/amp-customize-controls.asset.php';
+			$asset        = require $asset_file;
+			$dependencies = $asset['dependencies'];
+			$version      = $asset['version'];
+
 			wp_enqueue_script(
 				'amp-customize-controls',
 				amp_get_asset_url( 'js/amp-customize-controls.js' ),
-				[ 'jquery', 'customize-controls' ],
-				AMP__VERSION,
+				array_merge( $dependencies, [ 'jquery', 'customize-controls' ] ),
```

Comment:
Pretty sure this would work:

1. In webpack.config.js
```JS
const customizer = {
	...sharedConfig,
	entry: {
		'amp-customize-controls': './assets/src/customizer/amp-customize-controls.js',
		'amp-customize-preview': './assets/src/customizer/amp-customize-preview.js',
		'amp-customizer-design-preview': './assets/src/customizer/amp-customizer-design-preview.js',
	},

	plugins: [
		...sharedConfig.plugins,
		new WebpackBar( {
			name: 'Customizer',
			color: '#f27136',
		} ),
		new DependencyExtractionWebpackPlugin( {
			useDefaults: false,
			requestToExternal: ( request ) => {
				if ( 'customize-controls' === request ) {
					return request;
				}
				return defaultRequestToExternal( request );
			},
		} ),
	],
};
```

2. In amp-customize-controls.js

```
import jquery from 'jquery';
import customize from 'customize-controls';

// ...

}( customize, jquery ) ); // last line

```

3. Update this line to remove `[ 'jquery', 'customize-controls' ]` -- doesn't the dependency extraction Webpack plugin handle jQuery?

Then a no-unresolved ESlint exception for `customize-controls`.